src/users/integration.test.ts

```diff
@@ -79,13 +79,13 @@ requireAtLeast(
       await expectDave().toHaveProperty('active', true);
 
       // Second activation is a no-op
-      await expect(client.users.db.activate('dynamic-dave')).resolves.toEqual(true);
+      await expect(client.users.db.activate('dynamic-dave')).resolves.toEqual(false);
 
       await client.users.db.deactivate('dynamic-dave');
```

Comment:
can you add `.resolves.toEqual(true)`

Reply:
addressed in https://github.com/weaviate/typescript-client/pull/285/commits/7d9310ab0621162d4bde50c1c09bbf7c222ef58c